### ISSUE TITLE
Editing Recipes

### ISSUE THREAD

**Issue by pamreid01** [jlai/fitness-dashboard#67]
You have done an incredible job in creating this dashboard. Thank you. My only request is to have the ability to edit recipes. It isn't a huge thing since I can delete and input correctly using you dashboard which I can't do on the iPhone app.

Again thank,
Gabby

**Comment by jlai** [comment]
Hi Gabby, on the food page there's options to edit custom meals and custom foods. Is that what you're looking for or is there something else?

**Comment by pamreid01** [comment]
Hi, I am referring to the ability to change the ingredients in a custom recipe. If I want to add chia seeds to a smoothie, I don't see a way to do that. Am I missing something?

Thank you!

**Comment by mscatz-9** [comment]
Yes, you can do that.  Click on Food - > Meal -> Manage Meals -> Select Meal (that you want to customize) -> scroll to the bottom of the list -> Select a Food (this is where you can search for chia seeds) and be sure to add and update your meal.  Hope that made sense.

**Comment by pamreid01** [comment]
I think the problem is that I created these recipes while we had on computer dashboard, so all I put in was the name and nutritional values. There are no ingredients to edit. Sorry.  I will create some and try out your explanation

Thank you SO much!

**Comment by mscatz-9** [comment]
Yes, it sounds like you created "custom foods" where you list serving size, calories, nutritional info.

Custom meals is where you would create your smoothies, etc. and list all the components that go into it.

**Comment by pamreid01** [comment]
Thank you.


Sent from the all new AOL app for iOS


On Monday, September 9, 2024, 4:10 PM, mscatz-9 ***@***.***> wrote:




Yes, it sounds like you created "custom foods" where you list serving size, calories, nutritional info.

Custom meals is where you would create your smoothies, etc. and list all the components that go into it.
